migrator.go

```diff
@@ -48,6 +48,7 @@ var typeAliasMap = map[string][]string{
 	"numeric":                  {"decimal"},
 	"timestamptz":              {"timestamp with time zone"},
 	"timestamp with time zone": {"timestamptz"},
+	"bool":                     {"boolean"},
 }
```

Comment:
Since this is a simple map, you still need to add the `boolean` to `bool` mapping in reverse, although this is not elegant.

Reply:
Thanks, done